### ISSUE TITLE
Beatmaps having no backgrounds appear with a broken image icon on modder info page

### ISSUE THREAD

**Issue by zeusminus** [ppy/osu-web#7510]
I couldn't find any issue regarding this one, excuse me if it's a duplicate.

When a recent event regarding a beatmap with no background is listed on the modder info page (`/users/<userid>/modding`), the background portion appears to be a broken image icon (see the screenshot below) instead of something more appealing.

![](https://zeu.s-ul.eu/NJdW2Z5g.png)

**Comment by notbakaneko** [comment]
Was the image for this beatmap removed?

**Comment by zeusminus** [comment]
It must have been uploaded without a background image, I think. I don't know if the owner removed it themselves.

**Comment by peppy** [comment]
@zeusminus just to better understand what happened here, you marked the map as explicit based off the background being explicit? this may be a case of the background not being available for display *due to it being marked explicit*. I believe this is how we handle explicit backgrounds in other places.

**Comment by zeusminus** [comment]
No, I marked it because of the song, and I also marked another one that has a background (again, because the song was explicit), but it remained intact.

![Screenshot_20210423-100535~2.png](https://user-images.githubusercontent.com/52823712/115832653-cb7d7080-a41b-11eb-89f4-1863c4841727.png)

I'm guessing that it should probably be something related to the set having no background image, rather than the explicit marker.

**Comment by nanaya** [comment]
This has been fixed in #8791
